### ISSUE TITLE
[BUG] Plots are being saved without labels and titles.

### ISSUE THREAD

**Issue by melovv** [pycaret/pycaret#1780]
Plots are being saved without labels and titles.

**To Reproduce**

```python
pip install pycaret[full]

from pycaret.regression import *
from pycaret.datasets import get_data
data = get_data('insurance')

s = setup(data, target = 'charges', silent=True)

dt = create_model('dt')

plot_model(dt, plot='residuals', save=True)

```
![Residuals_bug](https://user-images.githubusercontent.com/19955538/139469534-432fb181-2cad-4046-893d-82bc9561f503.png)

**Expected behavior**
Plot saved with title and labels.

![Residuals_fixed](https://user-images.githubusercontent.com/19955538/139469557-34247f24-dfcf-4eb7-8e0b-4922380e65e8.png)

**Additional context**
How I fixed it:

Update /home/ec2-user/anaconda3/envs/python3/lib/python3.6/site-packages/pycaret/internal/plots/yellowbrick.py 

In line 118, replace
```python
plt.savefig(f"{plot_filename}.png") #, bbox_inches="tight")
```
with 

```python
visualizer.show(outpath=f"{plot_filename}.png")
```

**Versions**
<details>

Python 3.6.13 | packaged by conda-forge | (default, Feb 19 2021, 05:36:01) 
[GCC 9.3.0] on linux

pycaret.__version__ = 2.3.4
matplotlib @ file:///home/conda/feedstock_root/build_artifacts/matplotlib-suite_1611858699142/work
yellowbrick==1.3.post1

</details>

**Comment by melovv** [comment]
I can try, for sure.

**Comment by moezali1** [comment]
Hi @melovv, I have assigned this to you. Please feel free to submit PR.

**Comment by harsh204016** [comment]
Hi @moezali1, could I try to solve this issue ..

**Comment by Mr-Optimistic** [comment]
Has this been solved?

**Comment by holetz** [comment]
As this issue is still pending, I fixed it as suggested by @melovv and it's working as it should now.
Should I submit a PR?

**Comment by ngupta23** [comment]
Yes, please go ahead and submit a PR. We would appreciate it. Please branch from `develop` and submit the PR back to `develop` branch.

**Comment by moezali1** [comment]
Closing as part of the 4.0 backlog cleanup — see #4177.

PyCaret 4.0 is a ground-up rewrite (OOP-only API, sklearn-native engine, Python 3.11+, sklearn ≥ 1.7) and almost all pre-4.0 issues either reference removed APIs (`setup()`, `plot_model`, `evaluate_model`, `interpret_model`, `automl`, the legacy time-series module, the Streamlit dashboard, …) or no longer reproduce on the new code path.

**If this still affects you in 4.0, please re-file as a new issue** with:

- the experiment constructor call (`Experiment(...)`),
- the operation that fails,
- the full traceback,
- your Python / sklearn / pycaret versions.

Resources for 4.0:

- Docs: https://pycaret.org/docs
- Migrate from 3.x: https://pycaret.org/docs/guides/migrating-from-3-x
- API reference: https://pycaret.org/reference
- FAQ: https://pycaret.org/docs/resources/faqs

Thanks for filing — and apologies for the long latency. The repo had drifted; we're starting clean.
